src/routes/github/create-branch/github-remove-session.ts

```diff
@@ -0,0 +1,15 @@
+import { Request, Response } from "express";
+
+
+export const GithubRemoveSession = (req: Request, res: Response) => {
+	const { gitHubAppConfig } = res.locals;
+
+	if (!gitHubAppConfig) {
+		res.sendStatus(401);
+		return;
+	}
+
+	req.session.githubToken = undefined;
```

Comment:
Refreshing the page after this recreates my session, you might need to destroy the whole value?
		`req.session = null`

Reply:
This will not prevent to re-create the session on refresh.  You have explicitly sign out from the GitHub portal.